app/controllers/search_controller.rb

```diff
@@ -145,5 +131,19 @@ def fetch_consignments(queries)
       }
     end
   end
-end
 
+  def magma_error_wrapper
+    begin
+      magma_response = yield
+      if magma_response.code != '200'
```

Comment:
ibid, this code shouldn't run because the Magma::Client should raise upstream of this

Reply:
It runs, because Magma sends back 500's

Reply:
https://github.com/mountetna/magma/blob/dabad78e3aedaf53821ae9d1c7d7eb8ebb4182e5/gem/lib/client.rb#L114

Aha, the relevant line is commented out